src/utils/thread_helper.cpp

```diff
@@ -1281,6 +1317,9 @@ otError ThreadHelper::RetrieveTelemetryData(Mdns::Publisher *aPublisher, threadn
             infraLinkInfo->set_global_unicast_address_count(addressCounters.mGlobalUnicastAddresses);
         }
         // End of InfraLinkInfo section.
+
+        // NodeDivergenceInfo section
+        GetNodeDivergenceInfo(wpanBorderRouter->mutable_node_divergence_info());
```

Comment:
Please use same MACRO condition when invoking this function, i.e., add the OTBR_ENABLE_BORDER_ROUTING condition.

Reply:
Thanks!  The marco is already in Line #1302